substrate/cmd/bootstrap.go

```diff
@@ -56,5 +69,5 @@ func Apply(cmd *cobra.Command, args []string) {
 		logging.FromContext(ctx).Error(err.Error())
 		return
 	}
-	logging.FromContext(ctx).Infof("Applied substrate %s after %s", name, time.Since(start))
+	logging.FromContext(ctx).Infof("✅ Bootstrap finished, substrate name %s time taken %s", name, time.Since(start))
```

Comment:
nit: this log line doesn't as plain english, especially "name %s time taken %s". I could see something like "Bootstrapped %s after %d".